Jumpscale/builder/apps/BuilderFreeflow.py

```diff
@@ -25,7 +24,7 @@ def install(self, reset =False):
                                           "php-imagick"
                                          ])
 
-        #j.builder.tools.file_download("https://www.humhub.org/en/download/package/humhub-1.3.12.tar.gz","/var/www/html/humhub-1.3.12.tar.gz")
+        j.builder.tools.file_download("https://www.humhub.org/en/download/package/humhub-1.3.12.tar.gz","/var/www/html/humhub-1.3.12.tar.gz")
```

Comment:
you should check for the file first, otherwise it will raise an error if the file already exists

Reply:
done
will not raise error it just override the old one